encoder/vaapiencoder_base.cpp

```diff
@@ -51,6 +51,7 @@ VaapiEncoderBase::VaapiEncoderBase():
     m_videoParamCommon.frameRate.frameRateNum = 30;
     m_videoParamCommon.frameRate.frameRateDenom = 1;
     m_videoParamCommon.intraPeriod = 15;
+    m_videoParamCommon.ipPeriod = 1;
```

Comment:
ipPeriod = 1 means IP mode without B frame. That's default case. This name follows common codecs as I know.